### PR TITLE
V22 subtyping (1/N)

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
I am going to stagger these so the changes aren't so overwhelming. For this PR, I am running only GSVA (no changes) and then TP53:

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e99c13550956c95fa83e9c811515e04a01a8c3d4/scripts/run-for-subtyping.sh#L12-L16

#### What was your approach?
Ran the lines above, but had some errors in the TP53 module which required some code changes:
1. Remove https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e99c13550956c95fa83e9c811515e04a01a8c3d4/analyses/tp53_nf1_score/run_classifier.sh#L90-L92 and add this to the data release. This script was using `pbta-histologies.tsv`, and we really don't need to rerun if we keep the file from the previous step of generate data analysis files. See new PR to maintain this file #1432. It is also now included in #1365 
2. Update `focal-cn-file-preparation/02-add-ploidy-consensus.Rmd` to pull the above seg status file from `/data`
3. Add code around https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e99c13550956c95fa83e9c811515e04a01a8c3d4/analyses/tp53_nf1_score/run_classifier.sh#L114-L118 to skip these steps for subtyping, since the plots require fields created post-subtyping

#### What GitHub issue does your pull request address?
#1426 - partial

#### Which areas should receive a particularly close look?
code updates

#### Is there anything that you want to discuss further?
No, everything now runs as expected. File changes are minor.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### What types of results are included (e.g., table, figure)?
TP53 module file changes

#### What is your summary of the results?
as expected

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/molecular-subtyping-embryonal/00-embryonal-select-pathology-dx.R [hunk 1]
```diff
@@ -18,7 +18,8 @@ output_file <- file.path(root_dir,
 # as we identified in 00-v17-embryonal-select-pathology-dx.Rmd are:
 path_dx_terms<- c(
   "Supratentorial or Spinal Cord PNET",
-  "Embryonal Tumor with Multilayered Rosettes"
+  "Embryonal Tumor with Multilayered Rosettes (ETMR)",
+  "Embryonal Tumor with Multilayered Rosettes (ETMR);High-grade glioma/astrocytoma (WHO grade III/IV)"
```
Comment (jaclyn-taroni) [inline analyses/molecular-subtyping-embryonal/00-embryonal-select-pathology-dx.R:22]: Is _this_ expected?

Comment (jharenza) [reply analyses/molecular-subtyping-embryonal/00-embryonal-select-pathology-dx.R:22]: Yes, I added due to a new dx

Comment (jharenza) [reply analyses/molecular-subtyping-embryonal/00-embryonal-select-pathology-dx.R:22]: I can revert this back until I get to embryonal. I did it right away so I didn't forget about it, but let me revert that.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻

**Pull request merged**